.gitignore

```diff
@@ -38,3 +38,5 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+average_floor_difficulty_graph.png

```

Comment:
whats this?

Reply:
If you set `DEBUG` in constants to true, when you generate the dungeon you will get a floor difficulty graph. It was extremely useful for debugging so felt worth keeping